src/main/java/use_cases/ResetBalance/ResetBalanceInteractor.java

```diff
@@ -31,7 +31,7 @@ public void execute() {
 
         Transaction transaction = new TopupTransaction(curBalance + amountToAdd, new PricePoint(LocalDate.now(), curBalance));
 
-        super.addToHistory(userHistory, "Stonks App", user, curBalance + amountToAdd, curBalance, transaction);
+        super.addToHistory(userHistory, "Stonks App", curBalance, transaction);
```

Comment:
why are we adding to history on reset? I was under the impression that a reset clears everything, including resetting current balance to 10000, removing all owned stocks and clearing history?